VoterWeightPlugins/hooks/useTokenOwnerRecord.ts

```diff
@@ -0,0 +1,23 @@
+import {useUserCommunityTokenOwnerRecord, useUserCouncilTokenOwnerRecord} from "@hooks/queries/tokenOwnerRecord";
+import {TokenOwnerRecord, ProgramAccount} from "@solana/spl-governance";
+import {PublicKey} from "@solana/web3.js";
+
+export const useTokenOwnerRecord = (
```

Comment:
I could have sworn I wrote this exact hook some place but maybe not.

Reply:
There's `useTokenOwnerRecordByPubkeyQuery` which doesn't work here as we don't have the actual pubkey. Otherwise there's `useUserCommunityTokenOwnerRecord`and `useUserCouncilTokenOwnerRecord` that this hook uses.